docs/authoring content.md

```diff
@@ -34,7 +34,13 @@ Some common frontmatter fields that are natively supported by Quartz:
 - `draft`: Whether to publish the page or not. This is one way to make [[private pages|pages private]] in Quartz.
 - `date`: A string representing the day the note was published. Normally uses `YYYY-MM-DD` format.
 
+The frontmatter is parsed using the [gray-matter](https://github.com/jonschlinkert/gray-matter) library.
+
 ## Syncing your Content
 
 When your Quartz is at a point you're happy with, you can save your changes to GitHub.
 First, make sure you've [[setting up your GitHub repository|already setup your GitHub repository]] and then do `npx quartz sync`.
+
+## Customization
+
+Frontmatter parsing for `title`, `tags`, `aliases` and `cssclasses` is a functionality of the [[Frontmatter]] plugin. Frontmatter parsing for `date` is a functionality of the [[CreatedModifiedDate]] plugin. See the plugin pages for customization options.
```

Comment:
I think we need some way to link the features to the plugins that provide these features because the configuration is described there. When I read about the Syntax for the frontmatter in `features/authoring content.md` I would want to know to what extent I can change the behaviour (described in `plugin/Frontmatter` and `plugin/CreatedModifiedDate`).

Also this creates the backlinks for the plugin pages.

I'm open to suggestions how this linking could be better worded or otherwise achieved.

Reply:
you can just create a link to that page without the details, maybe something like `See also: ... for more details` or sth like that.

But it is also not a big deal, I don't have a strong opinion on this.

Reply:
sg.